spec/e2e/e2e_helper.rb

```diff
@@ -181,7 +181,7 @@ def install_ondemand
   if host_inventory['platform'] == 'redhat'
     release_rpm = "https://yum.osc.edu/ondemand/latest/ondemand-release-web-#{build_repo_version}-1.noarch.rpm"
     on hosts, "[ -f /etc/yum.repos.d/ondemand-web.repo ] || #{packager} install -y #{release_rpm}"
-    on hosts, "sed -i 's|/#{build_repo_version}/|/build/#{build_repo_version}/|g' /etc/yum.repos.d/ondemand-web.repo"
+    on hosts, "sed -i 's|ondemand/#{build_repo_version}/web|ondemand/build/#{build_repo_version}/web|g' /etc/yum.repos.d/ondemand-web.repo"
```

Comment:
Do we want to keep this?

Reply:
Yep, I made similar change to deb repo setup in different PR, just overlooked the EL repo having same change. I just noticed when running E2E tests and doing `BEAKER_destroy=no`.

Reply:
Oh i see the diff now, we were looking at `build` before too. Cool.